src/objects/deck/deck.stories.mdx

```diff
@@ -90,8 +100,32 @@ By default, the `o-deck` class will use CSS Grid Layout to arrange child element
 The `o-deck--align-start` modifier can be used to top-align deck items. This modifier is helpful when the deck items have different alignment. For example, the `o-media` object can be middle or top aligned depending on the length of the content. The modifier forces all deck objects to have the same alignment.
 
 <Canvas>
-  <Story name="Alignment" height="200px">
-    {(args) => alignmentDemo(args)}
+  <Story
+    name="Alignment"
+    height="200px"
+    args={{
+      columns: 3,
+      columnsBreakpoint: '@m',
+      class: 'o-deck--align-start',
+    }}
```

Comment:
Updated the alignment story to accept args, and passing in args to preserve the layout that was previously hard-coded.